deploy/cloudformation/elastic-agent-ec2.yml

```diff
@@ -60,6 +60,7 @@ Resources:
             - ec2:DescribeSnapshots
             - ec2:CreateSnapshots
             - ec2:CreateTags
+            - ec2:DeleteSnapshot
             Resource: "*"
```

Comment:
Not related to this pr, but shouldn't we have only ec2 resources here?

Reply:
you are right. it will be safer